AsyncDisplayKit/ASControlNode.m

```diff
@@ -170,6 +170,14 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
   if (!self.enabled)
     return;
 
+  // On iPhone 6s, iOS 9.2 (and maybe other versions) sometimes calls -touchesEnded:withEvent:
+  // on the view for one call to -touchesBegan:withEvent:. On ASControlNode, it used to trigger
+  // twice an action unintentionally. Now, we ignore that even if we're not in a tracking state
```

Comment:
Super minor grammar fix, if you do an update - this does NOT prevent landing lol!  "trigger an action twice"